.coveragerc

```diff
@@ -4,7 +4,12 @@
 omit =
     */test*
     */__init__.py
-source = qiita_core,qiita_ware,qiita_db,qiita_pet 
+
+    # these files will be ommited from coveralls because they contain test
+    # of the environment and methods they are executed regularly by travis,
+    # then there should not be concern over their "correctness".
+    qiita_core/environment_manager.py
+source = qiita_core,qiita_ware,qiita_db,qiita_pet
```

Comment:
There is, see also #120

On (Feb-13-15| 9:55), Joshua Shorenstein wrote:
> > @@ -4,7 +4,12 @@
> >  omit =
> >      */test*
> >      */__init__.py
> > -source = qiita_core,qiita_ware,qiita_db,qiita_pet 
> > +
> > +    # these files will be ommited from coveralls because they contain test
> > +    # of the environment and methods they are executed regularly by travis,
> > +    # then there should not be concern over their "correctness".
> > +    qiita_core/environment_manager.py
> > +source = qiita_core,qiita_ware,qiita_db,qiita_pet
> 
> Is there a way to test the click commands?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/853/files#r24682456